listenbrainz/db/user.py

```diff
@@ -560,3 +567,68 @@ def get_all_usernames():
         for row in result:
             user_id_map[row.id] = row.musicbrainz_id
     return user_id_map
+
+
+def pause(db_conn,id):
+    """ Sets the user's is_paused flag to true
+    with specified row ID from the database.
+    
+    Args:
+        db_conn: database connection
+        id (int): the row ID of the listenbrainz user
+    """
+    try:
+        db_conn.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET is_paused = true
+             WHERE id = :id
+            """), {
+            'id': id,
+        })
+        db_conn.commit()
+        _notify_user_email(db_conn,id,1)
+
+    except sqlalchemy.exc.ProgrammingError as err:
+        logger.error(err)
+        raise DatabaseException("Couldn't pause user: %s" % str(err))
+
+
+def unpause(db_conn,id):
+    """ Sets the user's is_paused flag to false
+    with specified row ID from the database.
+    
+    Args:
+        db_conn: database connection
+        id (int): the row ID of the listenbrainz user
+    """
+    try:
+        db_conn.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET is_paused = false
+             WHERE id = :id
+            """), {
+            'id': id,
+        })
+        db_conn.commit()
+        _notify_user_email(db_conn,id,0)
+
+    except sqlalchemy.exc.ProgrammingError as err:
+        logger.error(err)
+        raise DatabaseException("Couldn't pause user: %s" % str(err))
+
+
+def _notify_user_email(db_conn, user_id,paused):
+    user = get(db_conn, user_id, fetch_email=True)
+    if user["email"] is None:
+        return
+    url = current_app.config['SERVER_ROOT_URL']
+    template = 'emails/id_paused.txt' if paused else 'emails/id_unpaused.txt'
+    subject = 'ListenBrainz UserID Paused' if paused else 'ListenBrainz UserID Unpaused'
```

Comment:
I think the strings here should say:

"Your ListenBrainz account has been paused and is not accepting incoming listens"
and
"Your ListenBrainz account has been unpaused and is accepting incoming listens"

Options: Add the account name: "Your ListenBrainz accout ([account_name]) has been..."